core/src/main/scala/stryker4s/run/MutantRunner.scala

```diff
@@ -75,18 +75,24 @@ abstract class MutantRunner(process: ProcessRunner, sourceCollector: SourceColle
     filePath.overwrite(mutatedFile.tree.syntax)
   }
 
-  private def runMutants(mutatedFiles: Iterable[MutatedFile], tmpDir: File): Iterable[MutantRunResult] = {
+  private def runMutants(mutatedFiles: Iterable[MutatedFile],
+                         tmpDir: File): Iterable[MutantRunResult] = {
     val totalMutants = mutatedFiles.flatMap(_.mutants).size
+    val etaCalculator = new EtaCalculator(totalMutants)
```

Comment:
It's probably nicer to have this constructor injected, and in the test simply test if the function is called with the correct parameters. Then the EtaCalculator can test if the correct things are logged. IoC and all that

Reply:
I agree, but I'm not sure how I should handle passing `totalMutants` then, since it's not available at time of constructing. =/

Reply:
Perhaps a `start(totalMutants)` function that returns a new EtaCalculator or something...?